boards/sodaq_one/Cargo.toml

```diff
@@ -0,0 +1,53 @@
+[package]
+name = "sodaq_one"
+version = "0.1.0"
+authors = ["Robert Hennig <robert.hennig@freylax.de>"]
+description = "Board Support crate for the SODAQ ONE"
+keywords = ["no-std", "arm", "cortex-m", "embedded-hal"]
+license = "MIT OR Apache-2.0"
+repository = "https://github.com/atsamd-rs/atsamd"
+readme = "README.md"
+documentation = "https://atsamd-rs.github.io/atsamd/atsamd21g18a/sodaq_one/"
+
+[dependencies]
+cortex-m = "~0.5"
+embedded-hal = "~0.2"
+nb = "~0.1"
+
+[dependencies.cortex-m-rt]
+version = "~0.6"
+optional = true
+
+[dependencies.atsamd21-hal]
+path = "../../hal"
+version = "~0.2"
+default-features = false
+
+[dev-dependencies]
+panic-abort = "~0.3"
+panic-semihosting = "~0.5"
+cortex-m-semihosting = "~0.3"
+cortex-m-rtfm = "~0.3"
+sx1509 = "~0.2"
```

Comment:
you can remove this dep because none of your examples use this; please double check to make sure that you're only listing what you need!